package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@trycourier/courier",
-  "version": "3.2.1",
+  "version": "3.3.0",
```

Comment:
@cgradwohl mind rebasing and changing this to 3.4.0? I used 3.3.0 in bulk processing sorrey

Reply:
also can we update CHANGELOG?

Reply:
yeah sure